src/ui/Quickview/Quickview.ts

```diff
@@ -244,13 +245,36 @@ export class Quickview extends Component {
     // If there is no content inside the Quickview div,
     // we need to add something that will show up in the result template itself
     if (/^\s*$/.test(this.element.innerHTML)) {
+      const resultsSection = $$(this.root).find('.coveo-results-column');
```

Comment:
This is "somewhat" flaky, as it uses some arbitrary css class, which might *not* always be there in some special corner case.

Would it work if we simply used `this.root` as a "delimiter" ?
As long as the popup stays inside the search interface we should be good (does not have to be the result column), and `this.root` will always be defined.

Reply:
Okay perfect!